tests/spec_examples.rs

```diff
@@ -17,17 +19,18 @@ fn get_examples() -> Vec<String> {
         .collect()
 }
 
-use ethpm;
-
 // Try to parse packages located in
 // ethpm-spec/examples/**/*.json
 // TODO: Generate 1 case per example
 #[test]
 fn can_parse_examples() {
     for example in get_examples() {
+        // let raw_object: serde_json::Value = serde_json::from_str(&example).unwrap();
+        // println!("JSON: {}", raw_object);
+
         // Try to parse without failure
-        let package: ethpm::Package = serde_json::from_str(&example).unwrap();
-        let raw_object: serde_json::Value = serde_json::from_str(&example).unwrap();
-        // TODO iterate through `raw_object` and assert field match to `package`
+        let package: Package = serde_json::from_str(&example).unwrap();
+
+        println!("Package: {:?}", package);
```

Comment:
Still feel like there needs to be some sort of deeper testing of this functionality, although simply parsing is probably a decent test of the package parser working